hedera-mirror-rest/api/v1/openapi.yml

```
@@ -3145,3 +3174,77 @@ components:
         type: array
         items:
           type: string
+    contractsBlockNumberQueryParam:
+      name: block.number
+      in: query
+      description: The block's number
+      examples:
+        noValue:
+          summary: --
+          value: ""
+        noOperator:
+          summary: Example of block.number equals with no operator
+          value: 1
+        eqOperator:
+          summary: Example of block.number equals operator
+          value: eq:2
+        hexEncoded:
+          summary: Example of hex encoded block.number equals with no operator
+          value: '0xa0'
+        hexEncodedEqOperator:
+          summary: Example of hex encoded block.number equals with equals operator
+          value: eq:0xa0
+      schema:
+        type: string
+        pattern: ^((eq):)?(\d{1,19}|0x([a-fA-f0-9])+?)$
```

nit: Some unnecessary grouping
```suggestion
        pattern: ^(eq:)?(\d{1,19}|0x[a-fA-f0-9]+)$
```